src/datagrid/constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
```

Comment:
Follow the nameing pattern.  All files under this directory has prefix DataGrid... use same for this too.